sema4ai/vscode-client/src/extensionCreateEnv.ts

```diff
@@ -390,17 +390,28 @@ export async function basicValidations(
     }
     let canProceed: boolean = true;
     for (const failedCheck of rccDiagnostics.failedChecks) {
+        let func;
         if (failedCheck.status == STATUS_FATAL) {
             canProceed = false;
+            func = window.showErrorMessage;
         }
-        let func = window.showErrorMessage;
         if (failedCheck.status == STATUS_WARNING) {
-            func = window.showWarningMessage;
+            if (failedCheck.category == 1030 && failedCheck.type == "OS") {
+                // No pop-up but added to the output so that is visible in support logs
+                OUTPUT_CHANNEL.appendLine("RCC diagostics warning/info: " + failedCheck.message);
```

Comment:
I'm not sure about other statuses from RCC... aren't there errors too?

If so, the default should be `window.showErrorMessage` (as it was) and then here it could set `func = undefined` (so that we don't loose them).

Reply:
Good catch